### DISCUSSION TITLE
Cannot find "files changed" in pull request

### DISCUSSION THREAD

**Discussion by David263** [community/community#22351] [New to GitHub]
<p>I am taking one of the GitHub courses, on pull requests, and it keeps giving me the instruction "On the pull request, click <strong>Files changed." I cannot find any link or button with this caption! Please help.</strong></p>

**Comment by mpboom** [comment, marked as answer]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f609d26ae739713f553d82baf59414ac510100bd.png" data-download-href="/uploads/short-url/z6yv4pyBLCV4TxuVhXRTjPeE24Z.png?dl=1" title="files-changed.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/19948032/181070295-16a21aa6-4ba3-4cd6-bdef-c42999873ffe.png" alt="files-changed.PNG" data-base62-sha1="z6yv4pyBLCV4TxuVhXRTjPeE24Z" width="690" height="277" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/f609d26ae739713f553d82baf59414ac510100bd_2_690x277.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/f609d26ae739713f553d82baf59414ac510100bd_2_1035x415.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/f609d26ae739713f553d82baf59414ac510100bd.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f609d26ae739713f553d82baf59414ac510100bd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">files-changed.PNG</span><span class="informations">1099×442 39.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>Hi there, see the screenshot from the Atom repository. The button is right there. The URL it leads to looks like this:
<pre><code>https://github.com/atom/atom/pull/18678/files
</code></pre>
<p>or</p>
<pre><code>https://github.com/&lt;user&gt;/&lt;repo&gt;/pull/&lt;PR number&gt;/files</code></pre>
